### DISCUSSION TITLE
about PDF background and CSS properties

### DISCUSSION THREAD

**Discussion by ghost** [diegomura/react-pdf#1508] [Ideas]
hope you guys doing well after looking for too many projects and trying  them final got the  this perfect  package for my project to generate pdf 

is there any way to give background-color/background to the pdf-page that will cover entire page /pages...?

here's is  details what i mean..
.. actually  i have an query suppose i have different background Color for my pdf but when components ends  then it shows white space ( ) ex page height 1400px for a page and  components    that i wanna print  its height is  1800px with none white background .. so when it renders the 2nd page   1800 - 1400  so rest 400 px will print in next page so in 2nd page 1800 - 400  =  1400 px  (more then half of page )we will get white space (blank) ,... as there is not component left to render

![Capture](https://user-images.githubusercontent.com/53328265/133727459-0a6d1c3b-49a5-4dd7-bae4-c6de3a5399d2.PNG)

**Comment by jeetiss** [comment]
Here the fix #1541 for this problem

**Reply by ghost** [reply]
well i am not sure it will work with dynamic content (where page height is  dynamic) as well as multiple pages...hope you are getting my point...thanks for the suggestion btw

**Reply by jeetiss** [reply]
@patelrishi fix is released! can you approve that you problem with background is solved in `v2.0.21`?
